Sources/Misc/Obsoletions.swift

```diff
@@ -526,6 +526,115 @@ public extension Purchases {
         fatalError()
     }
 
+    /**
+     * Configures an instance of the Purchases SDK with a specified API key.
+     *
+     * The instance will be set as a singleton.
+     * You should access the singleton instance using ``Purchases/shared``
+     *
+     * - Note: Use this initializer if your app does not have an account system.
+     * ``Purchases`` will generate a unique identifier for the current device and persist it to `NSUserDefaults`.
+     * This also affects the behavior of ``Purchases/restorePurchases(completion:)``.
+     *
+     * - Parameter apiKey: The API Key generated for your app from https://app.revenuecat.com/
+     *
+     * - Returns: An instantiated ``Purchases`` object that has been set as a singleton.
+     */
+    @objc(configureWithAPIKey:)
+    @available(iOS, obsoleted: 1, message: """
+    If you're using observer mode with StoreKit 2, ensure that you call Purchases.handleObserverModeTransaction(result:) after making a purchase.
```

Comment:
this note is great, let's also add it to the docstring though. In the docstring we can even use double backticks syntax to create a link to the other method